spec/insertPreview.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Preview UI HTML', function() {
+
+  var InsertPreview = require('../examples/lib/insertPreview')
+  var sequencer = require('../src/ImageSequencer')()
+  var insertPreview;
+
+  beforeEach(()=>{
+    insertPreview = InsertPreview
+
+    spyOn(insertPreview,'generatePreview')
+    spyOn(insertPreview,'updatePreviews')
+
+    insertPreview.generatePreview()
```

Comment:
These functions take parameters. Testing params with `toHaveBeenCalledWith` might be good